services/ansible/ansible-quality.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+const { floorCount } = require('../../lib/color-formatters')
+
+const ansibleContentSchema = Joi.object({
+  quality_score: Joi.number().required(),
+}).required()
+
+class AnsibleGalaxyContent extends BaseJsonService {
+  async fetch({ projectId }) {
+    const url = `https://galaxy.ansible.com/api/v1/content/${projectId}/`
+    return this._requestJson({
+      url,
+      schema: ansibleContentSchema,
```

Comment:
Do we know what sort of HTTP error response codes we could get from this API? For example what happens on unknown/invalid project id, etc. If we know we'll get back certain responses (404 for example) in certain situations then we can add that to the request call:

```js
schema: ansibleContentSchema,
errorMessages: {
  404: 'project not found',
  403: 'foo',
}
```

Reply:
There are no custom error messages defined on the existing ansible badges: https://github.com/badges/shields/blob/d96d8ae193213ff6f6166071fde1a0638bed2633/services/ansible/ansible.service.js#L19-L27

Unless there's some specific condition we're expecting to handle on this endpoint, its OK to stick with the default error handling.

Reply:
Works for me. I mentioned it because the corresponding badge(s) `quality | not found` wasn't really clear to me, but I'm not a user of the service

Reply:
Maybe @calvinbui you can help us to clarify the best message as you open the issue?

Reply:
`No Score Available` like this one here: https://galaxy.ansible.com/s2504s/datadog